packages/devextreme/js/ui/tabs.js

```diff
@@ -346,13 +357,30 @@ const Tabs = CollectionWidget.inherit({
         return this._$wrapper;
     },
 
-    _renderScrollable: function() {
-        const $itemContainer = this.$element().wrapInner($('<div>').addClass(TABS_SCROLLABLE_CLASS)).children();
+    _getScrollableDirection() {
         const isVertical = this._isVertical();
-        const scrollableDirection = isVertical ? 'vertical' : 'horizontal';
+        const scrollableDirection = isVertical ? SCROLLABLE_DIRECTION.vertical : SCROLLABLE_DIRECTION.horizontal;
+
+        return scrollableDirection;
+    },
+
+    _updateScrollableDirection() {
+        const scrollable = this.getScrollable();
+
+        if(!scrollable) {
+            this._renderScrolling();
+        } else {
+            const scrollableDirection = this._getScrollableDirection();
+
+            scrollable.option('direction', scrollableDirection);
+        }
```

Comment:
```suggestion
        if(scrollable) {
            scrollable.option('direction', this._getScrollableDirection());
        } else {
            this._renderScrolling();
        }
```
the point here is why use 'not' here )

Reply:
[20cb1ef422683b209d85f6e3e12c47fff990f2d2]